src/panels/calendar/dialog-calendar-event-editor.ts

```diff
@@ -302,7 +302,11 @@ class DialogCalendarEventEditor extends LitElement {
     const duration = differenceInMilliseconds(this._dtend!, this._dtstart!);
 
     this._dtstart = new Date(
-      ev.detail.value + "T" + this._dtstart!.toISOString().split("T")[1]
+      ev.detail.value +
+        "T" +
+        this._dtstart!.toLocaleTimeString("en-GB", {
```

Comment:
We should create a helper for this :-) and also for the time in `en-CA`

Reply:
You mean locally in that class and not globally? I will put that on my list, but merge this PR for now to make sure we have that fix here in for the release today.